README.md

```diff
@@ -376,7 +376,7 @@ Supabase.initialize(
 
 The redirect callback url should have this format `[YOUR_SCHEME]://[YOUR_AUTH_HOSTNAME]`
 
-![authentication settings page](https://user-images.githubusercontent.com/689843/124574731-f735c180-de74-11eb-8f50-2d34161261dd.png)
+![authentication settings page](https://raw.githubusercontent.com/Wizzel1/supabase-flutter/5ba88b53b6459ab5420876b50275af316b8f62d2/screenshot.png)
```

Comment:
```suggestion
![authentication settings page](https://raw.githubusercontent.com/supabase/supabase-flutter/main/screenshot.png)
```

Reply:
@Vinzent03 the link you provided shows a 404

Reply:
That's because it's not yet merged. Once the file actually exists on the main branch it should work.